Lectures/Lecture7_Reproducibility.md

```diff
@@ -1,35 +1,104 @@
-# Open Science for Physicists, Lecture [number] , [main topic]
+# Open Science for Physicists, Lecture 7, Documentation for reproducibility
 
-*This is a template file, please make a copy in the ../Lectures director before customizing for a specific lecture*
-
-+ Last updated: DD MM 20YY
++ Last updated: 29 08 2024
 + Lecturer: e.g. @SanliFaez 
 
 #### Contributors: 
-+ Erik van Sebille @erikvansebille
-+ Hosein Nemati
++ Sjoerd Terpstra @TerpstraS
 
 ## Development goal
-__why are we covering this topic and how can this lecture help participants progress towards that learning goal __
+Obtaining an intrinsic motivatio to documenting the project for replication and reproducibility
+
+![Project documentation illustrated as a roadmap](reproducibility.jpg)
+[The Turing Way project](https://the-turing-way.netlify.app/reproducible-research/reproducible-research) illustration by Scriberia. Used under a CC-BY 4.0 licence. DOI: 10.5281/zenodo.3332807.
+
+Student teams start with their measurement projects. They will also define the goal of their experiment and the analysis procedures, and create the proper documentation folder.
```

Comment:
Since this week is about reproducing the earlier projects, maybe we can say something like this
```suggestion
Student teams aim to reproduce a project from their peers from week 4. They will also review the documentation of the project and their experiences with reproducing the experiment.
```